src/dc_receive_imf.rs

```diff
@@ -1617,13 +1555,17 @@ async fn create_group_record(
     chat_id
 }
 
+/// Creates ad-hoc group ID.
+///
+/// Algorithm:
+/// - sort normalized, lowercased, e-mail addresses alphabetically
+/// - put all e-mail addresses into a single string, separate the address by a single comma
+/// - sha-256 this string (without possibly terminating null-characters)
+/// - encode the first 64 bits of the sha-256 output as lowercase hex (results in 16 characters from the set [0-9a-f])
+///
+/// This ensures that different Delta Chat clients generate the same group ID unless some of them
+/// are hidden in BCC.
```

Comment:
Why does it matter that different clients generate the same group ID?  Is it sent out as group ID when replying or so?  I feel like this comment would be even better if it also clarified that.

Reply:
I originally removed this group ID generation but then realized it will result in split groups. Yes, it is sent when you reply from Delta Chat.